src/hooks/constants/graphql/Constants.graphql

```diff
@@ -0,0 +1,16 @@
+type Fee {
+  numerator: String!
+  denominator: String!
+}
```

Comment:
Please find another type name than `fee`, because we have various fees within this domain.

Reply:
This isn't applicable, since all runtime 'fee' types are in this format (or actually [u32,u32])